mhs/common/mhs_common/workflow/asynchronous_reliable.py

```diff
@@ -8,10 +8,17 @@
 class AsynchronousReliableWorkflow(common_asynchronous.CommonAsynchronousWorkflow):
     """Handles the workflow for the asynchronous reliable messaging pattern."""
 
-    async def handle_outbound_message(self, message_id: str, correlation_id: str, interaction_details: dict,
+    async def handle_outbound_message(self, from_asid: Optional[str],
+                                      message_id: str, correlation_id: str, interaction_details: dict,
                                       payload: str) -> Tuple[int, str]:
```

Comment:
Think this type hint needs to be updated:
```suggestion
                                       payload: str) -> Tuple[int, str, Optional[wd.WorkDescription]]:
```